main.py

```diff
@@ -300,8 +300,8 @@ def __add_zap_options__(self, args: List[str]) -> NoReturn:
         if self.auth_password_field_name is not None:
             zap_options.append(self.__add_custom_option__(label="auth.password_field", value=self.auth_password_field_name))
         
-        # ZAP options should be wrapped with ""
-        if len(zap_options) > 0:
+        # ZAP options should be wrapped with "" when automatic auth is enabled
+        if len(zap_options) > 0 and self.auth_loginUrl is not None:
```

Comment:
just to have it somewhere when passing --requestHeader it's necessary to wrap everything with single quotes.
eg: `--requestHeader="'authorization:Bearer tokenValue'"`